### ISSUE TITLE
Add the possibility to decide query projection parameters

### ISSUE THREAD

**Issue by alcheagle** [BlinkTagInc/node-gtfs#85]
I would suggest to add the possibility to pass a list for filtering the fields of mongo query. 

For example this:
```
exports.getAgencies = (query = {}) => {
  ...
  return Agency.find(query, '-_id').lean();
};
```

would become this:
```
exports.getAgencies = (query = {}, projection='-_id') => {
  ...
  return Agency.find(query, projection).lean();
};
```

**Comment by brendannee** [comment]
Great idea!

I implemented this on the master branch. https://github.com/BlinkTagInc/node-gtfs/commit/5a634b9514663c7b2afa35c7a912a0b9c4e1b4b3

It seems to work as intended, would you mind checking out the branch and letting me know what you think?

I also updated the README to include info about this - now nearly all methods support both `projection` and `options`. Please let me know if you have ideas on how to make this better.

I also added you to the contributors list in package.json.

**Comment by alcheagle** [comment]
Perfect, everything works as expected. 

I would add a link to ``mongoose`` documentation in the README, where ``projection`` and ``options`` fields are explained.

Thank you.

**Comment by brendannee** [comment]
I added links as you suggested and published this as 1.3.0 release.

Let me know if you have other ideas on how this library can be improve.
